src/PostgrestTransformBuilder.ts

```diff
@@ -344,4 +339,26 @@ export default class PostgrestTransformBuilder<
       Method
     >
   }
+
+  /**
+   * Set the maximum number of rows that can be affected by the query.
+   * Only available in PostgREST v13+ and only works with PATCH and DELETE methods.
+   *
+   * @param value - The maximum number of rows that can be affected
+   */
+  maxAffected(
+    value: number
+  ): ClientOptions['postgrestVersion'] extends 13
+    ? Method extends 'PATCH' | 'DELETE'
```

Comment:
@avallete This works for RPC too, could we add it that? The docs have been updated recently ([ref](https://docs.postgrest.org/en/v13/references/api/preferences.html#max-affected), see RPC at the bottom).

Reply:
Good catch didn't tested the `rpc` they do have their own kind of typing though because they're a bit specifics.

About the "set of" that's actually something I wanna work on. It will be related to "embedded relationships with functions". I have some draft I need to find the time to work on it.

Reply:
> For now I think runtime error is okay, but we'll surely be able to catch that at type level at some point indeed. Just wondering, what if the function doesn't returns a setof but just plain records ? Or return table ?

Right now it will just fail silently, but we're working on a fix to raise an error.

Reply:
Added a specific case for `RPC` calls as we can't really know the request method at type inference time, nor can we know if it returns a \`setof\` another table. So it'll be a bit of degraded experience typewise (more chances to have error only at runtime) but should be good enough for now.

Reply:
> Right now it will just fail silently, but we're working on a fix to raise an error.

JFYI, the above is now fixed and now an error will be raised: https://github.com/PostgREST/postgrest/releases/tag/v13.0.3